src/components/resetPassword/__snapshots__/resetPassword.test.jsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`should render without crashing 1`] = `ReactWrapper {}`;
```

Comment:
The component of this test is not rendered

Reply:
let me look into this